contrib/platform/src/com/sun/jna/platform/win32/Guid.java

```diff
@@ -411,7 +417,7 @@ public ByReference(GUID guid) {
              *            the memory
              */
             public ByReference(Pointer memory) {
-
+                super(memory);
```

Comment:
Good catch.   The the `Pointer`-based constructor is omitted, JNA still does the right thing (albeit less efficiently), but if the constructor is there, it better be correct.  Depending on how this is used, you may want to call `.read()` after the `super()` call.